dashboard/app/models/overflow_activities.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: activities
+#
+#  id              :integer          not null, primary key
+#  user_id         :integer
+#  level_id        :integer
+#  action          :string(255)
+#  url             :string(255)
+#  created_at      :datetime
+#  updated_at      :datetime
+#  attempt         :integer
+#  time            :integer
+#  test_result     :integer
+#  level_source_id :integer
+#  lines           :integer          default(0), not null
+#
+# Indexes
+#
+#  index_activities_on_level_source_id       (level_source_id)
+#  index_activities_on_user_id_and_level_id  (user_id,level_id)
+#
+
+class OverflowActivity < ActiveRecord::Base
```

Comment:
@wjordan noted offline that perhaps this should descend from `Activity`. If desired, I can play around with this, noting that doing so may introduce some complexity (does DB table name need to be specified? is there any other AR magic that needs to be configured?).